Entrega/exercicio-05.js

```diff
@@ -0,0 +1,15 @@
+window.alert("Medida diferente de ZERO");
```

Comment:
Acho que aqui compensaria também colocar dentro de uma condição, tipo se for menor ou igual a zero, nao verifica não aceita.